src/db/migrations/3_CreateConfigTable.js

```diff
@@ -0,0 +1,31 @@
+const debug = require("../debug");
+
+class CreateConfigTable extends require("../Migration") {
+  async body(index, database) {
+    let done = false;
+    let sql = await this.sql();
+
+    // await sql.schema.dropTableIfExists("test");
+
+    if (!(await sql.schema.hasTable("scraper_config"))) {
```

Comment:
Don't call it `scraper_config` because we have in the same db event scraper and twitter scraper. Call it explicitly `event_scraper_config`.